.github/workflows/ci.yml

```diff
@@ -20,8 +20,9 @@ jobs:
       - uses: actions/checkout@v4
       - name: install golang
         shell: bash
-        run: brew install go
-	run: make install-requirements
+        run: |
+           brew install go
+           make install-requirements
```

Comment:
This can only be done on Linux right now?

Reply:
Not if we use pipx...

Ubuntu 24.04 and macOS bail when using pip and recommend pipx

Reply:
This is why CI stopped running FWIW:

run:

followed by

run: